src/adapters/rubicon.js

```diff
@@ -152,22 +224,32 @@ function RubiconAdapter() {
 
   function handleRpCB(responseText, bidRequest) {
     let responseObj = JSON.parse(responseText); // can throw
+    var ads = responseObj.ads;
+
+    console.log('bid request', bidRequest);
+    console.log('response', responseObj);
 
-    if(
-      typeof responseObj !== 'object' ||
-      responseObj.status !== 'ok' ||
-      !Array.isArray(responseObj.ads) ||
-      responseObj.ads.length < 1
-    ) {
+    // check overall response
+    if(typeof responseObj !== 'object' || responseObj.status !== 'ok') {
       throw 'bad response';
     }
 
-    var ads = responseObj.ads;
+    // video ads array is wrapped in an object
+    if (bidRequest.mediaType === 'video') {
+      if(typeof ads === 'object') {
+        ads = ads[Object.keys(ads)[0]];
```

Comment:
Correct me if I'm wrong - prebid.js makes a separate request for each ad unit - in that case frank will return an object with the single ads response in it. Made a change here to grab the placement code since that is the name of the array in the ad object - I think this is more reliable :)